src/main/java/ro/itschool/ema/exceptions/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,15 @@
+package ro.itschool.ema.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
```

Comment:
I see a problem here which is that I have already made a controller advice class in my branch (ExceptionHandlerAdvice).

Reply:
they have different names so we can push the code and then have a meeting and refactor the code to keep the best of both sides